### PR TITLE
implement ubuntu checklist

### PR DESCRIPTION
**Pull request by fujexo**
This PR implements the Role for Ubuntu.

### CODE REVIEW

File: roles/maintenance_12_ubuntu/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
```
Comment (s3lph) [inline roles/maintenance_12_ubuntu/tasks/main.yml:12]:
```suggestion
    taskid: ignore-me
```
see https://github.com/adfinis/ansible-collection-maintenance/pull/11

File: roles/maintenance_12_ubuntu/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+
+ubuntu_release_train: lts
+# Define which version of Ubuntu the Upgrade is checked against
+# Possible options: all, devel, latest, lts, stable, supported, supported-esm, series, unsupported
+
+maintenance_exclude_tasks: []
+# Define defaults which can be overriden on a host-by-host basis
```
Comment (s3lph) [inline roles/maintenance_12_ubuntu/defaults/main.yml:8]: Any reasons these comments are **below** their respective subject?

**Review by s3lph — CHANGES_REQUESTED** [review]

File: roles/maintenance_12_ubuntu/defaults/main.yml (hunk 2 above)
Comment (fujexo) [reply roles/maintenance_12_ubuntu/defaults/main.yml]: Not in particular, fix it :)

**Review by s3lph — APPROVED** [review]
LGTM 🚀

**Pull request merged**